.teamcity/_self/projects/WPComTests.kt

```diff
@@ -49,8 +49,17 @@ object WPComTests : Project({
 	// Editor Tracking Edge
 	buildType(editorTrackingBuildType("desktop", "c752ca9a-e77d-11ec-8fea-0242ac120002", atomic=false, edge=true));
 
-	buildType(jetpackPlaywrightBuildType("desktop", "68fe6336-5869-4244-b236-cca23ba03487"));
-	buildType(jetpackPlaywrightBuildType("mobile", "a80b5c10-1fef-4c7f-9e2c-5c5c30d637c8"));
+	// Jetpack for Connected Non-WPCOM Sites
+	buildType(jetpackPlaywrightBuildType("desktop", "68fe6336-5869-4244-b236-cca23ba03487", "not-wpcom"));
```

Comment:
non-blocking: we could use named parameters here to make it a bit more readable. E.g. `jetpackTarget ="not-wpcom"`. (https://kotlinlang.org/docs/functions.html#named-arguments)

Reply:
Fixed in [6acd928](https://github.com/Automattic/wp-calypso/pull/73457/commits/6acd928c6cfc133d195d5578dc07e44821eef125)